lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -23,6 +23,6 @@ num-traits = "0.2.14"
 openssl = { version = "0.10.38", features = ["vendored"] }
 rand = { version = "0.8.5", features = ["getrandom"] }
 rand_chacha = "0.3.1"
-rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "f51222e7edeaf2954e2c02db3506aa36e02304ca" }
+rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "e0a659cd05edb13b64dd8735d7df59c61f1d39fa" }
```

Comment:
TODO(isaiah): This should be changed to the merge commit once https://github.com/gravitational/rdp-rs/pull/9 is merged.